concurrency.go

```diff
@@ -8,3 +8,18 @@ func Async[T any](f func() T) chan T {
 	}()
 	return ch
 }
+
+// Async0 executes a function in a goroutine and returns a channel closed after execution.
+func Async0(f func()) chan struct{} {
+	ch := make(chan struct{})
+	go func() {
+		f()
```

Comment:
Why not sending a message into channel, like we did for `Async`?

```go
f()
ch <- struct{}{}
close(ch)
```

Reply:
No particular reason.

That might be enough then WDYT?

```go
f()
ch <- struct{}{}
```